src/d/actor/d_a_npc_km1.cpp

```diff
@@ -4,248 +4,816 @@
 //
 
 #include "d/actor/d_a_npc_km1.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_s_play.h"
+#include "d/d_snap.h"
 #include "m_Do/m_Do_ext.h"
 #include "d/d_procname.h"
 #include "d/d_priority.h"
+#include "weak_bss_936_to_1036.h"
+static daNpc_Km1_HIO_c l_HIO;
 
 /* 000000EC-00000144       .text __ct__15daNpc_Km1_HIO_cFv */
 daNpc_Km1_HIO_c::daNpc_Km1_HIO_c() {
-    /* Nonmatching */
-}
 
-/* 00000144-000002F0       .text nodeCallBack_Km__FP7J3DNodei */
-static BOOL nodeCallBack_Km(J3DNode*, int) {
-    /* Nonmatching */
+    static daNpc_Km1_c::prm_tbl a_prm_tbl[] = {0x1FFE38E0, 0xE002C720, 0x00000000, 0x00000000, 
+        0x08000800, 150.0, 0x00000000};
```

Comment:
What exactly is happening here? The prm tbl has 12 fields but there are only 7 entries in this struct initializer. Furthermore, most of these values don't fit, `0x1FFE38E0` is too large to be an `s16` for example. Am I missing something here?